src/common/__tests__/CommonUtils.test.ts

```diff
@@ -268,6 +268,38 @@ describe('CommonUtils', () => {
         ]);
     });
 
+    // Disabling this test for now. It runs & passes locally but it fails
```

Comment:
As part of this PR I added this new test to increase coverage but as I've stated in the comments, while it passes locally, it fails in CI. I decided to keep the code here instead of throwing it out hoping that in the future we might be able to figure out why this is happening.